tests/_support/Database/Migrations/20160428212500_Create_test_tables.php

```diff
@@ -91,6 +91,101 @@ public function up()
 		]);
 		$this->forge->addKey('id', true);
 		$this->forge->createTable('misc', true);
+		
+		if (($this->db->DBDriver === 'SQLite3') || ($this->db->DBDriver === 'MySQLi')) 
+		{
+		  //Database Type test table
+		  //missing types : 
+		  //TINYINT,MEDIUMINT,BIT,YEAR,BINARY , VARBINARY, TINYTEXT,LONGTEXT,YEAR,JSON,Spatial data types
+		  $this->forge->addField([
+			'id'    => [
+				'type'          => 'BIGINT',
+				'constraint'    => 20,
+				$unique_or_auto => true,
+			],
+			'type_varchar'   => [
+				'type'          => 'VARCHAR',
+				'constraint'    => 40,
+				'null'          => true,
+			],
+			'type_char'   => [
+				'type'          => 'CHAR',
+				'constraint'    => 10,
+				'null'          => true,
+			],
+			'type_enum'   => [
+				'type'          => 'ENUM',
+				'constraint'    => ['appel','pears','bananas'],
+				'null'          => true,
+			],
+			'type_set'   => [
+				'type'          => 'SET',
+				'constraint'    => ['one','two'],
+				'null'          => true,
+			],
+			'type_text' => [
+			       'type'           => 'TEXT',
+			       'null'           => true,
+			],
+			'type_mediumtext' => [
+			      'type'            => 'MEDIUMTEXT',
+			      'null'            => true,
+			],
+			'type_smallint'   => [
+			      'type'            => 'SMALLINT',
+			      'null'            => true,
+			],
+			'type_integer'   => [
+			      'type'            => 'INTEGER',
+			      'null'            => true,
+			],
+			'type_float'   => [
+			      'type'            => 'FLOAT',
+			      'null'            => true,
+			],
+			'type_real'   => [
+			      'type'            => 'REAL',
+			      'null'            => true,
+			],
+			'type_double'   => [
+			      'type'            => 'DOUBLE',
+			      'null'            => true,
+			],
+			'type_decimal'   => [
+			     'type'             => 'DECIMAL',
+			     'constraint'       => '18,4',
+			     'null'             => true,
+			],
+			'type_numeric'   => [
+			     'type'             => 'NUMERIC',
+			     'constraint'       => '18,2',
+			     'null'             => true,
+			],
+			'type_date'   => [
+			     'type'             => 'DATE',
+			     'null'             => true,
+			],
+			'type_time'   => [
+			     'type'             => 'TIME',
+			     'null'             => true,
+			],
+			
+			'type_datetime'   => [
+			     'type'             => 'DATETIME',
+			     'null'             => true,
+			],
+			'type_timestamp'   => [
+			     'type'             => 'TIMESTAMP',
+			     'null'             => true,
+			],
+			'type_blob'   => [
+			     'type'             => 'BLOB',
+			     'null'             => true,
+			],
```

Comment:
Please align this entire segment inside `if` correctly.

Reply:
Will copy and paste to notepad wokring in the browser is bit of hassle.

Reply:
Copied and tried to algin values etc as above and below